internal/cloudapi/v2/errors.go

```diff
@@ -144,24 +146,43 @@ func find(code ServiceErrorCode) *serviceError {
 			return &e
 		}
 	}
-	return &serviceError{ErrorServiceErrorNotFound, http.StatusInternalServerError, "Error does not exist"}
+	return &serviceError{ErrorServiceErrorNotFound, http.StatusInternalServerError, "Error does not exist", ""}
 }
 
 // Make an echo compatible error out of a service error
 func HTTPError(code ServiceErrorCode) error {
-	return HTTPErrorWithInternal(code, nil)
+	return HTTPErrorWithInternal(code, nil, "")
 }
 
 // echo.HTTPError has a message interface{} field, which can be used to include the ServiceErrorCode
-func HTTPErrorWithInternal(code ServiceErrorCode, internalErr error) error {
+func HTTPErrorWithInternal(code ServiceErrorCode, internalErr error, details string) error {
```

Comment:
To reduce the diff a bit can we just have `HTTPErrorWithDetails` ? Rename this function to httpError (with all fields, not calitapized), and then keep the HTTPErrorWithInternal signature the same?